src/Forms/TextCheckboxGroupField.php

```diff
@@ -103,7 +103,8 @@ public function performReadonlyTransformation()
             if ($showTitle) {
                 $field->replaceField('ShowTitle', LiteralField::create(
                     'ShowTitle',
-                    $showTitle->Value() === 'Yes' ? $displayedText : $notDisplayedText
+                    // Intentionally using getValue() instead of getFormattedValue() to get the raw value
+                    $showTitle->getValue() === 'Yes' ? $displayedText : $notDisplayedText
```

Comment:
Why do we want to check against the raw value here? `ShowTitle` is a `CheckboxField` which will by default use ints `1` and `0` (but probably allows other value types) so this condition will often fail.

Reply:
Good spot, updated

Reply:
The "but probably allows other value types" part of my comment also worries me a little. Presumably it deals in truthy and falsy, not explicitly 1 and 0? If I'm right about that, we should make this a loose truthy check, not an exact equals check.

Reply:
While CheckboxField::getValue() has a mixed return type, it does simply return 1 or 0.  I think it's safe to rely on that not being changed by a subclass as there's no real reason to change it. If you want we could remove the === 1 bit of the comparison but I don't think it's necessary

Reply:
Right you are - sorry, I was conflating `$value` with `getValue()`.